website/blog/2018-04-25-fxa-train-110.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Firefox Accounts Train-110
+author: Vlad Filippov
+authorURL: https://twitter.com/vladikoff
+---
+
+This week we shipped FxA train-110 to production,
+with the following highlights:
+
+
+
+<!--truncate-->
+
+
+
+## FxA-141: lockbox beta
+
+We have enabled email-first flow for OAuth reliers.
+
+See https://github.com/mozilla/fxa-content-server/pull/6034 for details
+
+## FxA-143: MFA
+
+We made improvements to the TOTP (2FA) flow.
+These improvements deal with update properties of TOTP, such as recovery codes, emails and rate-limiting.
```

Comment:
The text in this section doesn't flow as well as the others.

Maybe something along the lines of:
`We made several improvements to the TOTP (2FA) flow: increased recovery code length, better emails, and improved rate limiting.`